apps/webservice/src/app/[workspaceSlug]/SidebarWorkspaceDropdown.tsx

```diff
@@ -27,7 +27,7 @@ export const SidebarWorkspaceDropdown: React.FC<{ workspace: Workspace }> = ({
 }) => {
   const { data } = useSession();
   const workspaces = api.workspace.list.useQuery();
-  const update = api.profile.update.useMutation();
+  const update = api.profile.updateActiveWorkspace.useMutation();
```

Comment:
why? Don't you need to pass in a workspaceId?

Reply:
Updating the username has nothing to do with the workspace. I can change it back if you want.

Reply:
This was a side effect of me changing the api endpoint.